nlg/search.py

```diff
@@ -194,6 +196,27 @@ def search_quant(self, quants, nround=2, cell_fmt='df["{}"].iloc[{}]'):
             self.results[tk] = {
                 'location': 'cell', 'tmpl': cell_fmt.format(self.df.columns[y], x),
                 'type': 'quant'}
+            
+    def search_derived_quant(self, quants, nround=2):
+        """Search the common derived dataframe parameters for a set of quantitative values.
+
+        Parameters
+        ----------
+        quants : list / array like
+            The values to search.
+        nround : int, optional
+            Numeric values in the dataframe are rounded to these many
+            significant digits before searching.
+        """
+        dfclean = utils.sanitize_df(self.df, nround)
+        quants = np.array(quants)
+        n_quant = quants.astype('float').round(2)
```

Comment:
What is this for?

Reply:
This is not needed now. As discussed we will enclose this in the new Template interface